### PR TITLE
Add key-points to the aggregation episode.

### PR DESCRIPTION
**Pull request by bsmith89**
Summarized some of the important functions, keywords, and concepts.

References #129 

Feedback appreciated on choice of key points.

### CODE REVIEW

File: _episodes/02-sql-aggregation.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ objectives:
 - "Filter and order results of a query based on aggregate functions."
 - "Save a query to make a new table."
 - "Apply filters to find missing values in SQL."
+keypoints:
+- "Use the `GROUP BY` keyword to aggregate data."
+- "Functions like `MIN`, `MAX`, `AVERAGE`, `SUM`, `COUNT`, etc. operate on aggreggated data."
```
Comment (ashander) [inline _episodes/02-sql-aggregation.md]: spelling: extra 'g' in aggregated

File: _episodes/02-sql-aggregation.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ objectives:
...
+- "Use the `HAVING` keyword to filter on aggregate properties."
+- "Use a `VIEW` to save a query as though it were a new table."
```
Comment (ashander) [inline _episodes/02-sql-aggregation.md]: I might be more explicit and instead of "save a query as though it were a new table" I might say "save a query and later access it as though it were another table". But maybe that's a bit wordy.

Comment (bsmith89) [inline _episodes/02-sql-aggregation.md]: @ashander How about 'Use a `VIEW` to access the result of a query as though it was a new table.'?

Slightly less wordy; focuses on the access of the results rather than the saving.

File: _episodes/02-sql-aggregation.md (hunk 1 above)
Comment (bsmith89) [inline _episodes/02-sql-aggregation.md]: Thanks; good catch!

File: _episodes/02-sql-aggregation.md (hunk 2 above)
Comment (ashander) [inline _episodes/02-sql-aggregation.md]: That's great. :+1: from me

**Comment by tracykteal** [comment]
thanks for the PR and feedback. Also agree these look good.
